extensions/usage-metrics-dashboard/CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing to Usage Metrics Dashboard
+
+## Organization
+
+- The app's core code lives in `app.R`
+- Supporting functions live in `R/`. Code in `R/` can be tests.
```

Comment:
What do you mean by "Code in `R/` can be tests." below you say that tests live in `tests/testthat`

Reply:
Oops, this was a typo.